unified-runtime/source/loader/layers/tracing/ur_tracing_layer.cpp

```diff
@@ -40,6 +40,10 @@ static std::shared_ptr<XptiContextManager> xptiContextManagerGet() {
   static auto contextManager = std::make_shared<XptiContextManager>();
   return contextManager;
 }
+
+// The Unified Runtime API calls are meant to be performant and creating an
+// event for each API Call will add significant overheads.
+static xpti_td *GURCallEvent = nullptr;
 static thread_local xpti_td *activeEvent;
```

Comment:
We can just remove activeEvent variable if every trace call will use the exact same one.